src/booking.js

```diff
@@ -426,7 +426,7 @@ class RecrasBooking {
 
             if (product.aantal < packageLine.product.minimum_aantal) {
                 this.setMinMaxAmountWarning(input.id, packageLine.product.minimum_aantal, 'minimum');
-            } else if (product.aantal > packageLine.max) {
+            } else if (packageLine.max > 0 && product.aantal > packageLine.max) {
```

Comment:
shouldn't we distinguish between `packageLine.max === 0` and `packageLine.max === undefined || packageLine.max === null`? `max === 0` sounds a lot like 'the maximum amount you can book here is `0`', i.e. 'you may not book this item'

Reply:
Good point, will change.